floor_generator/lib/processor/error/query_method_writer_error.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:analyzer/dart/element/element.dart';
+import 'package:source_gen/source_gen.dart';
+
+class QueryMethodWriterError {
+  final MethodElement _methodElement;
+
+  QueryMethodWriterError(final MethodElement methodElement)
```

Comment:
A bit of a personal opinion, but you might prefer having a trailing coma in the argument to format this better.